<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedback::Analyzer::SurveyEditor</class>
 <widget class="QWidget" name="UserFeedback::Analyzer::SurveyEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="surveyView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddSurvey">
   <property name="text">
    <string>&amp;Add Survey...</string>
   </property>
   <property name="toolTip">
    <string>Add new survery.</string>
   </property>
  </action>
  <action name="actionDeleteSurvey">
   <property name="text">
    <string>&amp;Delete Survey</string>
   </property>
   <property name="toolTip">
    <string>Delete a survey.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
